src/app/home/home-routing.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
```

Comment:
Not sure if this file is actually used as we have this module to be loaded at the beginning by default?